conduit/data/datasets/tabular/base.py

```diff
@@ -21,37 +32,28 @@ def __init__(
         x: Union[Tensor, npt.NDArray[np.floating], npt.NDArray[np.integer]],
         y: Optional[TargetData] = None,
         s: Optional[TargetData] = None,
-        transform: Optional[TabularTransform] = None,
-        target_transform: Optional[TabularTransform] = None,
-        cont_indexes: Optional[List[int]] = None,
-        disc_indexes: Optional[List[int]] = None,
+        non_ohe_indexes: Optional[List[int]] = None,
         feature_groups: Optional[List[slice]] = None,
-        target_groups: Optional[List[slice]] = None,
     ) -> None:
         if isinstance(x, np.ndarray):
             x = torch.as_tensor(x, dtype=torch.float32)
         super().__init__(x=x, y=y, s=s)
-        self.transform = transform
-        self.target_transform = target_transform
-
-        if self.transform is not None:
-            self.x = self.transform(self.x)
-        if self.target_transform is not None and self.y is not None:
-            self.y = self.target_transform(self.y)
 
-        self.cont_indexes = cont_indexes
-        self.disc_indexes = disc_indexes
+        self.non_ohe_indexes = non_ohe_indexes
         self.feature_groups = feature_groups
-        self.target_groups = target_groups
 
     @property
-    def x_cont(self) -> Optional[Tensor]:
-        if self.cont_indexes is None:
+    def x_non_ohe(self) -> Optional[Tensor]:
+        if self.non_ohe_indexes is None:
             return None
-        return self.x[:, self.cont_indexes]
+        return self.x[:, self.non_ohe_indexes]
 
-    @property
-    def x_disc(self) -> Optional[Tensor]:
-        if self.disc_indexes is None:
-            return None
-        return self.x[:, self.disc_indexes]
+    def fit_transform_(self, transform: TabularNormalize) -> None:
+        """Fit a transformation to the non-one-hot-encoded features and transfrom in-place."""
+        if (x_non_ohe := self.x_non_ohe) is not None:
```

Comment:
Use `some` from ranzen?

Reply:
:+1: 

(I also just spotted a typo in the docstring.)